assets/airports/cyow.json

```diff
@@ -0,0 +1,1769 @@
+                ["N45d11m48.00", "W075d26m57.00"],
+                ["N45d11m19.00", "W075d27m31.00"],
+                ["N45d10m52.00", "W075d28m08.00"],
+                ["N45d10m26.00", "W075d28m47.00"],
+                ["N45d10m02.00", "W075d29m27.00"],
+                ["N45d09m39.00", "W075d30m10.00"],
+                ["N45d09m17.00", "W075d30m54.00"],
+                ["N45d08m58.00", "W075d31m39.00"],
+                ["N45d08m40.00", "W075d32m26.00"],
+                ["N45d08m23.00", "W075d33m14.00"],
+                ["N45d08m09.00", "W075d34m03.00"],
+                ["N45d07m56.00", "W075d34m54.00"],
+                ["N45d07m46.00", "W075d35m45.00"],
+                ["N45d07m37.00", "W075d36m37.00"],
+                ["N45d07m30.00", "W075d37m29.00"],
+                ["N45d07m25.00", "W075d38m22.00"],
+                ["N45d07m22.00", "W075d39m16.00"],
+                ["N45d07m21.00", "W075d40m09.00"],
+                ["N45d07m22.00", "W075d41m02.00"],
+                ["N45d07m25.00", "W075d41m56.00"],
+                ["N45d07m30.00", "W075d42m49.00"],
+                ["N45d07m37.00", "W075d43m41.00"],
+                ["N45d07m46.00", "W075d44m33.00"],
+                ["N45d07m56.00", "W075d45m24.00"],
+                ["N45d08m09.00", "W075d46m15.00"],
+                ["N45d08m23.00", "W075d47m04.00"],
+                ["N45d08m40.00", "W075d47m52.00"],
+                ["N45d08m58.00", "W075d48m39.00"],
+                ["N45d09m17.00", "W075d49m24.00"],
+                ["N45d09m39.00", "W075d50m08.00"],
+                ["N45d10m02.00", "W075d50m51.00"],
+                ["N45d10m26.00", "W075d51m31.00"],
+                ["N45d10m52.00", "W075d52m10.00"],
+                ["N45d11m19.00", "W075d52m47.00"],
+                ["N45d11m48.00", "W075d53m21.00"],
+                ["N45d12m18.00", "W075d53m54.00"],
+                ["N45d12m49.00", "W075d54m24.00"],
+                ["N45d13m21.00", "W075d54m52.00"],
+                ["N45d13m54.00", "W075d55m17.00"],
+                ["N45d14m28.00", "W075d55m40.00"],
+                ["N45d15m03.00", "W075d56m01.00"],
+                ["N45d15m38.00", "W075d56m19.00"],
+                ["N45d16m14.00", "W075d56m34.00"],
+                ["N45d16m51.00", "W075d56m47.00"],
+                ["N45d17m28.00", "W075d56m56.00"],
+                ["N45d18m05.00", "W075d57m04.00"],
+                ["N45d18m42.00", "W075d57m08.00"],
+                ["N45d19m20.00", "W075d57m09.00"],
+                ["N45d19m57.00", "W075d57m08.00"],
+                ["N45d20m35.00", "W075d57m04.00"],
+                ["N45d21m12.00", "W075d56m58.00"],
+                ["N45d21m37.00", "W075d56m51.00"],
+                ["N45d23m07.00", "W075d59m03.00"],
+                ["N45d26m30.00", "W075d53m49.00"]
+            ]
+        }
+    ],
+    "arrivalRunway": "25",
+    "departureRunway": "14",
+    "fixes": {
+        "ADVOS": ["N45d20m19.00", "W075d23m90.00"],
```

Comment:
I don't know where you got the coordinates for all of the fixes but they seem to be just slightly off. Maybe try SkyVector?